VogtFactory/Dodge.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace VogtFactory
+{
+    class Dodge: RentalCar
+    {
+        public string name;
```

Comment:
I would make these constants and define the value directly on the variable. It doesnt sound like the name or number can ever change.

```c#
public const string Name = "minivan";
public const int MaxSeats = 7;
```

Reply:
Also, in c# public class variables should always be Pascal Case (i.e. first AlwaysCapitalizeFirstOfEveryWord)